e2e/testcases/cli_test.go

```diff
@@ -742,13 +742,15 @@ func TestNomosHydrateAndVetDryRepos(t *testing.T) {
 			sourceFormat:    string(configsync.SourceFormatUnstructured),
 			expectedOutPath: "../testdata/hydration/compiled/helm-overlay",
 		},
-		{
-			name:            "hydrate a DRY repo with remote base",
-			path:            "../testdata/hydration/remote-base",
-			outPath:         "remote-base/compiled",
-			sourceFormat:    string(configsync.SourceFormatUnstructured),
-			expectedOutPath: "../testdata/hydration/compiled/remote-base",
-		},
+		// remote-base relies on external public helm chart, which occasionally
```

Comment:
Will the removal of the remote-based test case result in decreased test coverage? The case seems to be passing fine in the post submit CI.

Reply:
> The case seems to be passing fine in the post submit CI.

It's not passing fine in the periodics. You can see that it's very flaky as it is in the presubmits: https://testgrid.k8s.io/googleoss-kpt-config-sync-main#kind

> Will the removal of the remote-based test case result in decreased test coverage?

Yes this means less test coverage, although this is more-so verifying kustomize functionality than Config Sync functionality.